salt/modules/zypperpkg.py

```diff
@@ -1367,6 +1366,17 @@ def refresh_db(force=None, root=None):
 
         {'<database name>': Bool}
 
+    gpgautoimport : False
+        If set to True, automatically trust and import public GPG key for
+        the repository.
+
+        .. versionadded:: 3005
+
+    repo
```

Comment:
I would suggest to rename the param to `repos` as there could be more than one initially.

Reply:
I used "repo" as yumpkg also use it. But it seems they can really do only 1.
But they have other options like "enablerepo" which can get a list and has the same effect.
As it is not consistent at all, we can also call it "repos" :-)